packages/core/admin/package.json

```diff
@@ -108,8 +108,8 @@
     "qs": "6.10.1",
     "react": "^17.0.2",
     "react-copy-to-clipboard": "^5.1.0",
-    "react-dnd": "^14.0.2",
-    "react-dnd-html5-backend": "^14.0.0",
+    "react-dnd": "15.1.2",
+    "react-dnd-html5-backend": "15.1.2",
```

Comment:
Do you think we need to pin the dependency? It hasn't created problems so far ...

Reply:
We do because you can't upgrade it further than this without having react18. I mean, they released it as a breaking change so it wouldn't auto update but still.

Reply:
Alright, thanks. That's good to know :/